onnx_example.py

```diff
@@ -15,7 +19,7 @@
 smiles_cols = ["SMILES"]
 y_cols = ["ACTIVITY"]
 x_cols = ["X1", "X2"]
-featurizer = MordredDescriptors()
+featurizer = MACCSKeysFingerprint()
```

Comment:
This works now with other featurizers?

Reply:
yes, i have uploaded models with mordred and MACCSKeysFingerprint till now